### PR TITLE
Add projects section to homepage

### PR DESCRIPTION
**Pull request by alexander-azizi-martin**
This PR:
- adds the project section to the homepage
- changes background color of sections on the homepage
- fixes spacing issues on the homepage

### CODE REVIEW

File: src/pages/index.md [hunk 1]
```diff
@@ -36,6 +36,28 @@ sections:
...
             style: secondary
+      - title: Projects
+        bgcolor: e8edf3
+        justify-content: flex-start
+        cards:
+          - image: /images/Rectangle-19.png
+            alt: Phone taking screenshot of Nike hoodie
+            label:  Project A
+            caption: Web Development
+            redirect: http://localhost:8888/projects
```
Comment (kandloic) [inline src/pages/index.md:47]: should the redirect not be: `/projects`?

File: src/pages/index.md [hunk 2]
```diff
@@ -36,6 +36,28 @@ sections:
...
+          - image: /images/Rectangle-19.png
+            alt: Ipad and coffee cup on table
+            label: Project B
+            caption: Web Development
+            redirect: http://localhost:8888/projects
```
Comment (kandloic) [inline src/pages/index.md:52]: see comment above

File: src/pages/index.md [hunk 3]
```diff
@@ -36,6 +36,28 @@ sections:
...
+          - image: /images/Rectangle-19.png
+            alt: Light bulb being held up in front of the horizon
+            label:  Project C
+            caption: Web Development
+            redirect: http://localhost:8888/projects
```
Comment (kandloic) [inline src/pages/index.md:57]: see comment above

File: src/pages/index.md [hunk 4]
```diff
@@ -36,6 +36,28 @@ sections:
...
+        link:
+          title: See All Projects
+          url: http://localhost:8888/
```
Comment (kandloic) [inline src/pages/index.md:60]: think url should just be: `/`

File: src/pages/index.md (hunk 1 above)
Comment (kandloic) [reply src/pages/index.md:47]: read more on absolute vs relative urls: https://docs.microsoft.com/en-us/sql/ado/guide/data/absolute-and-relative-urls?view=sql-server-ver16

**Review by kandloic — CHANGES_REQUESTED** [review]
Looks good overall. Left a few comments that should be addressed (:

**Comment by snhumphr** [comment]
This branch/pull request is now obsolete, the projects section update has now been merged into  https://github.com/Software-For-Love/software-for-love-site/tree/SFLW-65-section-1-home-page-redesign.

**Pull request closed without merging**